src/commands/shane/permset/create.ts

```diff
@@ -98,28 +98,31 @@ export default class PermSetCreate extends SfdxCommand {
             label: this.flags.name
         });
 
-        const objectList: Set<string> = new Set<string>();
+        let objectList: Array<string> = new Array<string>();
 
         if (!this.flags.object) {
             const files = fs.readdirSync(targetLocationObjects);
-            files.forEach(file => objectList.add(file));
+            files.forEach(file => objectList.push(file));
         } else {
-            objectList.add(this.flags.object);
+            objectList.push(this.flags.object);
         }
 
+        // Make sure there is no duplicate
+        objectList = Array.from(new Set(objectList));
+
         this.ux.log(`Object list is ${objectList}`);
 
         if (this.flags.checkpermissionable) {
             conn = this.org.getConnection();
 
             this.ux.startSpinner('Getting objects describe from org');
 
-            if (objectList.has('Activity')) {
+            if (objectList.includes('Activity')) {
                 // Describe call doesn't work with Activity, but works with Event & Task
                 // Both of them can be used for fieldPermissions
-                objectList.delete('Activity');
-                objectList.add('Event');
-                objectList.add('Task');
+                objectList.includes('Activity');
```

Comment:
@FabienTaillon this line I don't understand

Reply:
That's because it's a mistake ^^ I switched from Set to Array, so had to change the related methods. But what I want is to delete 'Activity' value, so it's definitely not what I'm doing here.

Reply:
If you know how to fix my TS error with Set and iterations, I can update the PR.

Reply:
I'm not sure what the TS issues are.  I assume all your config looks like mine, and you've run yarn install to get the latest stuff.

I google a bit for the set iteration ts issue, but they were all *really old*.

Reply:
The strange thing is that I was using Set for the previous PR and I had no error, can't find why I have now.